docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/heroku.md

```diff
@@ -236,16 +236,22 @@ Create a new `server.js` in a new [env](/developer-docs/latest/setup-deployment-
 
 ```js
 module.exports = ({ env }) => ({
-  url: env('MY_HEROKU_URL'),
-});
+    proxy: true,
+    url: env('MY_HEROKU_URL'),
+    app: { 
+      keys: env.array('APP_KEYS')
+    },
+  })
 ```
 
-You will also need to set the environment variable in Heroku for the `MY_HEROKU_URL`. This will populate the variable with something like `https://your-app.herokuapp.com`.
+You will also need to set the environment variable in Heroku for the `MY_HEROKU_URL` and `APP_KEYS`. This will populate the variables with something like `https://your-app.herokuapp.com/` and `dsfhasbvvfwfcerterzer+n1w==,afjdsagfsauzuwzref6==,kjdbgjerhgh6wireg==,jkssdhgjaksdgkjbsdg==` respectively.
 
 ```bash
 heroku config:set MY_HEROKU_URL=$(heroku info -s | grep web_url | cut -d= -f2)
+heroku config:set APP_KEYS=$(cat .env | grep APP_KEYS | cut -d= -f2-)
```

Comment:
why is there an extra `-` after `-f2` ?

Reply:
You may test this using the following script 
`bash
echo "HOST=0.0.0.0
PORT=1337
APP_KEYS=jksdhfasdjgut323t==,jaghuiererg7e4==,bjhwefbzuwe37r8==
JWT_SECRET=dfasjkdf7-sdfsadf78-bfwegfweo63-duhefw78
API_TOKEN_SALT=klasdngklnadsgnksa346u89345jhdfb7" | grep APP_KEYS | cut -d= -f2-
jksdhfasdjgut323t==,jaghuiererg7e4==,bjhwefbzuwe37r8==`

Reply:
thank you, I didn't know!